frontend/shared/src/components/Layout.tsx

```diff
@@ -13,7 +13,7 @@ const Main = styled.main`
 
 const Layout = ({ children }: { children: React.ReactNode }): JSX.Element => (
   <Main>
-    <Heading>Kesaseteli</Heading>
+    <Heading>employer</Heading>
```

Comment:
Capitalized?

Reply:
This is still more like a hard coded "hello world" text that will be replaced later with translations etc.